inscriptions/ISic000336.xml

```diff
@@ -35,8 +35,8 @@
      	       <resp>standardisation of template and tidying up encoding</resp>
  	   </respStmt>
 	    <respStmt>
-                    <name xml:id="system">system</name>
-                    <resp>automated or batch processes</resp>
+	        <name xml:id="GC" ref="https://orcid.org/0000-0003-1030-5679">Giorgia Capra</name>
```

Comment:
I think you've deleted the system respStmt and replaced it with your own? Please could you restore the system respStmt (it actually is there for a reason) and simply *add* your own resp statement either before or after it?